README.md

```diff
@@ -307,6 +328,18 @@ public interface ExampleHandler {
 }
 ```
 
+You also can make your handler extended from some other handler class by
```

Comment:
I thinks It's better to state that the only classes supported are ManualHandler and ExecutableHandler to avoid misunderstandings.